esp-hal-common/src/rtc_cntl/mod.rs

```diff
@@ -1,4 +1,70 @@
-//! Low-power Management
+//! RTC_CNTL (Real-Time Clock Control)
```

Comment:
I'm not sure which title is better. This module isn't entirely about the peripheral, but most of the sleep code is (or will be) implemented here.